### DISCUSSION TITLE
Show the detached filters in a related model

### DISCUSSION THREAD

**Discussion by msassa** [optimistdigital/nova-detached-filters#41] [Ideas]
Hi, I have a question, is posible to show this card in the detail page of a related model?
For example, I added this card to a resource called Cars.
Each car belong to a brand, that is another resource.
When I go to the brand detail, I see all the cars that belong to that brand, but I can't see the detached filters.
Is this posible?
Thanks

**Comment by KasparRosin** [comment]
No, it's not possible at this moment.
I'll transfer the issue to a discussion, where we can continue the discussion on this idea.
